networking/hardware_networks/uninstalling-sriov-operator.adoc

```diff
@@ -0,0 +1,50 @@
+[id="uninstalling-sriov-operator"]
+= Uninstalling the SR-IOV Network Operator
+include::modules/common-attributes.adoc[]
+:context: uninstalling-sr-iov-operator
+
+toc::[]
+
+To uninstall the SR-IOV Network Operator, you must first delete any running SR-IOV workloads. You must also delete the webhooks after uninstalling the operator.
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster using an account with `cluster-admin` permissions.
+* You have the SR-IOV Network Operator installed.
+
+.Procedure
+
+. Delete all SR-IOV custom resources (CRs).
+
+. Remove the SR-IOV Network Operator from your cluster by following the documentation on xref:../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
```

Comment:
"Remove the SR-IOV Network Operator from your cluster by following the documentation on deleting Operators from a cluster." -> "Follow the instructions on Deleting operators from a cluster to remove the SR-IOV Network Operator from your cluster."